PhysicsTools/NanoAOD/python/muons_cff.py

```diff
@@ -33,6 +33,14 @@
 )
 run2_miniAOD_80XLegacy.toModify(ptRatioRelForMu, srcLep = "slimmedMuonsUpdated")
 
+FSRforMu = cms.EDProducer("MuonFSRProducer",
+    packedPFCandidates = cms.InputTag("packedPFCandidates"),
+    slimmedMuons = cms.InputTag("slimmedMuons"),
+    slimmedElectrons = cms.InputTag("slimmedElectrons"),
+)
+run2_miniAOD_80XLegacy.toModify(FSRforMu, src = "slimmedMuonsUpdated")
+run2_nanoAOD_94X2016.toModify(FSRforMu, src = "slimmedMuonsUpdated")
```

Comment:
we only use slimmedMuonsUpdated for 80X, not for 94X2016 (see the rest of the file)